guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -96,6 +96,9 @@ endif::[]
 // Reclaiming Database Space
 include::common/modules/proc_reclaiming-postgresql-space.adoc[leveloffset=+3]
 
+// Updating Templates, Parameters, Lookup Keys and Values
+include::topics/post_upgrade_template_checks.adoc[leveloffset=+3]
+
 ifdef::katello,orcharhino,satellite[]
 //Tuning Satellite Server with Predefined Profiles
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+3]
```

Comment:
I think the Bugzilla was essentially that this should be:
```suggestion
include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
```

I'm not sure what made you think the old bits needed to be reintroduced.

Reply:
We just needed to check. Thanks Ewoud.

Reply:
No, it§s not desirable to change the leveloffset, because it's under the post-upgrade tasks.